src/EditorFeatures/VisualBasicTest/ExtractInterface/ExtractInterfaceTests.vb

```diff
@@ -843,8 +848,7 @@ Interface ISomeInterface2(Of T, U)
 End Interface</text>.NormalizedValue()
             Dim expectedCode = <text>
 Class Program(Of T, U)
-    Implements ISomeInterface(Of T), ISomeInterface2(Of T, U)
-    Implements IProgram(Of T, U)
+    Implements ISomeInterface(Of T), ISomeInterface2(Of T, U), IProgram(Of T, U)
```

Comment:
this is nice.  waht ended up causing this change?

Reply:
Running the formatter in a standard way on all changes for extract interface.

Reply:
Nice.  Love it!